### PR TITLE
Improve button style

### PR DESCRIPTION
**Pull request by mauricioabreu**
Old button had no style at all. This patch updates to a better style
(like the rest of the site).

### CODE REVIEW

**Comment by mauricioabreu** [comment]
You can see the final result in the attached image below:

speakerfight_better_reset_password_button

**Comment by luanfonceca** [comment]
Hi @mauricioabreu, really thanks for that. o/

Do you think it is possible for you to fix all the layout of this page, within this PR?!

You can use the `{% bootstrap_form form layout="inline" %}` and the same Titles and Subtitles layout that we have on the [login page](http://speakerfight.com/accounts/login/)?

If you need some help on how to do that, see the [login template](https://github.com/luanfonceca/speakerfight/blob/master/core/templates/account/login.html) or come and join us on the [Gitter chat](http://gitter.im/luanfonceca/speakerfight).

**Comment by mauricioabreu** [comment]
Of course! :-) Give me more time. I am working on another speakerfight PR.
I think the build failed for some permission reasons. Maybe doing what Travis suggested.

```
[speakerfight.com] Login password for 'root': 
No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.

tl;dr #

Not using sudo? Containers sound cool? Add sudo: false to travis.yml and you’re set.
```

**Comment by mauricioabreu** [comment]
Should I close this PR and open another?

**Comment by luanfonceca** [comment]
hmm i don't think it is needed, my only concern about this PR is that you made it from your master branch, it is not a good practice, because if your branch was not merged your code will have some commits that doesn't exist on the `origin` repository. As i said, for me, you can proceed and add the code here, without bigger problems. 

o/

**Comment by mauricioabreu** [comment]
I don't think it is a good/bad practice. I uses master because it is the main branch. :) But I got what you meant. 

Ok, I will use this same PR. Thank you!

**Comment by mauricioabreu** [comment]
Closing in favor to #84
